### DISCUSSION TITLE
Running MIFitHMM with tracks from Animotum

### DISCUSSION THREAD

**Discussion by mitchrider** [bmcclintock/momentuHMM#166] [General]
Hello,

I'm wondering if there is a way to use MIFitHMM with tracks reconstructed by Animotum to account for the measurement error of the predicted locations. 

Thank you!
Mitch

**Comment by bmcclintock** [comment]
Yes, in theory `MIfitHMM` will accept any list composed of `momentuHMMData` objects returned by `prepData`. So you'd just need to run `prepData` on each simulated track from {aniMotum} and then combine them into a list for `MIfitHMM`. I've not done this myself yet, but I was hoping to eventually include an example demonstrating exactly this in the {momentuHMM} vignette. So please let me know if you have any difficulties!

**Comment by mitchrider** [comment]
Thanks for getting back to me! I was able to do this with simulations from the `sim_fit` function in animotum. However, I realize those simulations don't constrain the tracks like `crwSimulator`  and  `crwPostIS` so I'm assuming I need to figure out a way to format that animotum output so I can use those functions to create simulations? Have you been able to do this?

**Reply by bmcclintock** [reply]
Are you just fitting the (discrete-time approximation of the) continuous-time correlated random walk (CTCRW) model in {aniMotum}? If so, maybe you should just use {crawl} to fit the CTCRW (`crawl::crwMLE`) and simulate from it (`crawl::crwSimulator` --> `crawl::crwPostIS`). Note that `momentuHMM::crawlWrap` is just a wrapper for these three {crawl} functions. Otherwise, have you looked into the {ctmm} package? I haven't looked into too much yet, but I *think* you can fit the CTCRW ("OUF" in {ctmm} parlance) and draw realizations of the observed path via simulation.
